docs/porting-guide.rst

```diff
@@ -1,23 +1,35 @@
 Porting guide
 =============
 
-xxx
----------------------
+PyModule_AddObject
+------------------
 
-`PyModule_AddObject()`: replace with a regular `HPy_SetAttr_s()`.  There is no `HPyModule_AddObject()` because it has an unusual refcount behaviour (stealing a reference but only when it returns 0).
+``PyModule_AddObject()`` is replaced with a regular ``HPy_SetAttr_s()``. There
+is no ``HPyModule_AddObject()`` because it has an unusual refcount behaviour (
+stealing a reference but only when it returns 0).
```

Comment:
`(stealing` was split into `( stealing`.

Reply:
shouldn't it be `"PyModule_AddObject is replace **by** a regular HPy_SetAttr_s?